lusSTR/scripts/filter_settings.py

```diff
@@ -28,20 +28,57 @@ def get_filter_metadata_file():
 
 def filters(locus_allele_info, locus, locus_reads, datatype, brack_col):
     metadata = filter_marker_data[locus]
-    if len(locus_allele_info) == 1:
-        locus_allele_info = single_allele_thresholds(metadata, locus_reads, locus_allele_info)
+    if locus == "AMELOGENIN":
+        locus_allele_info = filter_amel(metadata, locus_allele_info, locus_reads)
```

Comment:
There's nothing necessarily wrong with how you've updated this code, but we could keep the nesting complexity to a minimum with something like this, right?

```python
if locus == "AMELOGENIN":
   # ...
elif len(locus_allele_info) == 1:
   # ...
else:
   # ...
```

Reply:
I was just trying to avoid have to repeat code (line 34 would have to be in both blocks).